apps/condo/domains/marketplace/schema/fields/price.js

```diff
@@ -0,0 +1,86 @@
+const Ajv = require('ajv')
+const addFormats = require('ajv-formats')
+
+const { render, getGQLErrorValidator } = require('@condo/domains/common/schema/json.utils')
+const { ERROR_INVALID_PRICE } = require('@condo/domains/marketplace/constants')
+const PRICE_GQL_TYPE_NAME = 'PriceSchemaField'
+const PRICE_GQL_INPUT_NAME = 'PriceSchemaFieldInput'
+
+const priceSchemaFields = {
+    type: 'String!',
+    group: 'String!',
+    name: 'String!',
+    price: 'String!',
+    isMin: 'Boolean!',
+    vat: 'String!',
+    salesTax: 'String!',
```

Comment:
Hmm, looks like it's better to store percent values here
```suggestion
    vatPercent: 'String!',
    salesTaxPercent: 'String!',
```
Absolute values will appear within the Invoice model.